### PR TITLE
More MP FT 4.0 Repeat FATs

### PR DESCRIPTION
**Pull request by tevans78**

### CODE REVIEW

**Comment by tevans78** [comment]
#build

File: dev/com.ibm.ws.microprofile.faulttolerance_repeat_tests/src/com/ibm/ws/microprofile/faulttolerance/fat/repeat/RepeatFaultTolerance.java [hunk 1]
```diff
@@ -67,7 +67,7 @@ public static FeatureReplacementAction ft11metrics20Features(String server) {
      * @return the RepeatTests rule
      */
     public static RepeatTests repeatDefault(String server) {
-        return repeat(server, TestMode.FULL, MicroProfileActions.MP40, MicroProfileActions.MP20);
+        return repeat(server, TestMode.FULL, MicroProfileActions.MP50, MicroProfileActions.MP40, MicroProfileActions.MP20);
     }
```
Comment (Azquelt) [inline dev/com.ibm.ws.microprofile.faulttolerance_repeat_tests/src/com/ibm/ws/microprofile/faulttolerance/fat/repeat/RepeatFaultTolerance.java:88]: Please update the javadoc for the methods you've changed in this file.

File: dev/com.ibm.ws.microprofile.faulttolerance.cdi_fat_metrics/fat/src/com/ibm/websphere/microprofile/faulttolerance/metrics/fat/tests/circuitbreaker/CircuitBreakerMetricTest.java [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2019 IBM Corporation and others.
+ * Copyright (c) 2019, 2021 IBM Corporation and others.
```
Comment (Azquelt) [inline dev/com.ibm.ws.microprofile.faulttolerance.cdi_fat_metrics/fat/src/com/ibm/websphere/microprofile/faulttolerance/metrics/fat/tests/circuitbreaker/CircuitBreakerMetricTest.java]: Spurious date change?

File: dev/com.ibm.ws.microprofile.faulttolerance.cdi_fat/build.gradle [hunk 3]
```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2017, 2020 IBM Corporation and others.
+ * Copyright (c) 2017, 2021 IBM Corporation and others.
```
Comment (Azquelt) [inline dev/com.ibm.ws.microprofile.faulttolerance.cdi_fat/build.gradle:2]: Spurious date change?

**Review by Azquelt — CHANGES_REQUESTED** [review]

File: dev/com.ibm.ws.microprofile.faulttolerance.cdi_fat/build.gradle (hunk 3 above)
Comment (tevans78) [reply dev/com.ibm.ws.microprofile.faulttolerance.cdi_fat/build.gradle:2]: Was recently changed to add the JakartaTransformer

File: dev/com.ibm.ws.microprofile.faulttolerance_repeat_tests/src/com/ibm/ws/microprofile/faulttolerance/fat/repeat/RepeatFaultTolerance.java (hunk 1 above)
Comment (tevans78) [reply dev/com.ibm.ws.microprofile.faulttolerance_repeat_tests/src/com/ibm/ws/microprofile/faulttolerance/fat/repeat/RepeatFaultTolerance.java:88]: done

**Comment by tevans78** [comment]
#build

**Review by Azquelt — APPROVED** [review]

**Comment by tevans78** [comment]
#build

**Comment by tevans78** [comment]
#run-libby-bot

**Pull request merged**